doc/developers.rst

```diff
@@ -18,13 +18,23 @@ with more coverage.
 
 Building Documentation
 ----------------------
+The following is needed in order to build the documentation.
+
+*    ``pip install sphinx``
+*    ``pip install sphinx-gallery``
+*    ``pip install Image``
```

Comment:
reading through the `sphinx-gallery` docs, it seems you need `pillow`: https://github.com/sphinx-gallery/sphinx-gallery not `Image`.